engine/access/rest/converter.go

```diff
@@ -506,14 +506,31 @@ func idsResponse(ids []flow.Identifier) []string {
 	return res
 }
 
-func collectionResponse(flowCollection *flow.LightCollection, link LinkGenerator) generated.Collection {
-	self, _ := selfLink(flowCollection.ID(), link.CollectionLink)
+func collectionResponse(
+	collection *flow.LightCollection,
+	txs []*flow.TransactionBody,
+	link LinkGenerator,
+	expand map[string]bool,
+) generated.Collection {
+	self, _ := selfLink(collection.ID(), link.CollectionLink)
```

Comment:
why not throw the error back? like in other xxxResponse() functions?

Reply:
Yes, that will be a good improvement, sorry